filter/expand_json.go

```diff
@@ -10,11 +10,32 @@ import (
 	"github.com/jmespath/go-jmespath"
 )
 
+const expandJSONhelp = `
+ExpandJSON extracts values from a JSON formatted record field and writes them into other fields of the same record.
+It supports [JMESPath](https://jmespath.org/tutorial.html) to select the values to copy inside the JSON.
+
+### Example
+
+A possible filter configuration is:
+
+	[[filter]]
+	name="ExpandJSON"
+		[filter.config]
+		Source = "json_data"
+		[filter.config.Fields]
+		jfield1  = "field1"
+		jfield2  = "field2"
+		
+In this example, the filter will extract the value of the key ` + "`jfield1`" + ` and ` + "`jfield2`" + ` of the JSON 
```

Comment:
Generally using present tense in technical writings makes for _lighter_ and easier to read sentences. 
```suggestion
In this example, the filter extracts values of the ` + "`jfield1`" + ` and ` + "`jfield2`" + ` keys of the JSON 
```